src/scripts/1-create-requests.ts

```diff
@@ -0,0 +1,167 @@
+  console.log(`current network: ${getNetworkName()}`);
+
+  const runner = signer as unknown as ContractRunner;
+
+  // The modules are defined to be used in the sdk as known modules
+  const httpRequestModule = {
+    module: new Module(address.deployed[getNetworkName()].HTTP_REQUEST_MODULE, IHttpRequestModule.abi, runner),
+    abi: IHttpRequestModule.abi,
+  };
+
+  const bondedResponseModule = new Module(
+    address.deployed[getNetworkName()].BONDED_RESPONSE_MODULE,
+    IBondedResponseModule.abi,
+    runner
+  );
+
+  const bondedDisputeModule = {
+    module: new Module(address.deployed[getNetworkName()].BONDED_DISPUTE_MODULE, IBondedDisputeModule.abi, runner),
+    abi: IBondedDisputeModule.abi,
+  };
+
+  const arbitratorModule = {
+    module: new Module(address.deployed[getNetworkName()].ARBITRATOR_MODULE, IArbitratorModule.abi, runner),
+    abi: IArbitratorModule.abi,
+  };
+
+  const callbackModule = {
+    module: new Module(address.deployed[getNetworkName()].CALLBACK_MODULE, ICallbackModule.abi, runner),
+    abi: ICallbackModule.abi,
+  };
+
+  const knownModules: ModulesMap = {
+    [httpRequestModule.module.moduleAddress]: httpRequestModule.module,
+    [address.deployed[getNetworkName()].BONDED_RESPONSE_MODULE]: bondedResponseModule,
+    [bondedDisputeModule.module.moduleAddress]: bondedDisputeModule.module,
+    [arbitratorModule.module.moduleAddress]: arbitratorModule.module,
+    [callbackModule.module.moduleAddress]: callbackModule.module,
+  };
+
+  // The sdk is initialized with the runner, the oracle address and the known modules
+  const sdk = new ProphetSDK(runner, address.deployed[getNetworkName()].ORACLE, knownModules);
+
+  // usdt is going to be used as the bond token
+  const { usdt } = await getERC20Tokens();
+
+  const accountingExtension = await getAccountingExtension();
+
+  const deadline = FINALIZE_SOON ? getCurrentEpochInSeconds() + 30 : getRandomDeadline();
+
+  // Define the data to be sent for each module
+  const requestModuleData = Object.values({
+    url: 'https://defi.sucks/',
+    body: 'test body',
+    method: 0,
+    accountingExtension: address.deployed[getNetworkName()].ACCOUNTING_EXTENSION,
+    paymentToken: await usdt.getAddress(),
+    paymentAmount: BOND_SIZE,
+  });
+
+  const responseModuleData = Object.values({
+    accountingExtension: address.deployed[getNetworkName()].ACCOUNTING_EXTENSION,
+    bondToken: await usdt.getAddress(),
+    bondSize: BOND_SIZE,
+    deadline: deadline,
+    disputeWindow: 5000,
+  });
+
+  const disputeModuleData = Object.values({
+    accountingExtension: address.deployed[getNetworkName()].ACCOUNTING_EXTENSION,
+    bondToken: await usdt.getAddress(),
+    bondSize: BOND_SIZE,
+  });
+
+  const resolutionModuleData = [address.wallets.ARBITRATOR];
+
+  const finalityModuleData = Object.values({
+    target: address.opUniv3Pools[getNetworkName()][0],
+    data: '0x1a686502', // liquidity() selector
+  });
```

Comment:
Fetching a pool's state at the end of request might confuse the users. Let's drop the finality module, wdyt?